oak/module/placeholders.h

```
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2019 The Project Oak Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef OAK_MODULE_PLACEHOLDERS_H_
+#define OAK_MODULE_PLACEHOLDERS_H_
+
+#include <math.h>
+#include <pthread.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <sys/stat.h>
+#include <sys/types.h>
+#include <time.h>
+#include <unistd.h>
+
+// These placeholders were originally added to remove imports from the TensorFlow Lite.
+extern "C" {
+
+int __syscall5(int, int) { return NULL; }
```

Would it be better to let them `abort` or something that is more obvious, rather than silently return zeros?

---

Done